test/start-typescript.test.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const util = require('util')
+const { once } = require('events')
+const fs = require('fs')
+const crypto = require('crypto')
+const baseFilename = `${__dirname}/fixtures/test_${crypto.randomBytes(16).toString('hex')}`
+
+const t = require('tap')
+const test = t.test
+const sgetOriginal = require('simple-get').concat
+const sget = (opts, cb) => {
+  return new Promise((resolve, reject) => {
+    sgetOriginal(opts, (err, response, body) => {
+      if (err) return reject(err)
+      return resolve({ response, body })
+    })
+  })
+}
+const start = require('../start')
+
+// FIXME
+// paths are relative to the root of the project
+// this can be run only from there
```

Comment:
what does this mean? Can you create an issue or fix it here?

Reply:
this is present in start.test.js too, sure, I'll fix it.